plugins/utils/better-escape.nix

```
@@ -28,19 +28,17 @@ in
       Clear line after escaping if there is only whitespace.
     '';
 
-    keys =
-      helpers.defaultNullOpts.mkNullable (with types; either str helpers.nixvimTypes.rawLua) "<ESC>"
-        ''
-          Keys used for escaping, if it is a function will use the result everytime.
+    keys = helpers.defaultNullOpts.mkStr "<ESC>" ''
+      Keys used for escaping, if it is a function will use the result everytime.
 
-          Example (recommended):
+      Example (recommended):
 
-          keys.__raw = \'\'
-            function()
-              return vim.api.nvim_win_get_cursor(0)[2] > 1 and '<esc>l' or '<esc>'
-            end
-          \'\';
-        '';
+      keys.__raw = '''
```

```suggestion
      keys.__raw = '''
```
Why 3 simple quotes here ?

---

Within a double-quote indented string, triple-quotes is the correct way to render a double-quote.

The previous code rendered as `\'\'`.

---

Well, [it was ok here](https://nix-community.github.io/nixvim/plugins/better-escape.html#pluginsbetter-escapekeys), but this was definitely necessary elsewhere where I made this change. Maybe being within a markdown code block makes a difference.